Changes in github.com/digitalocean/go-libvirt prevent build

The package `github.com/digitalocean/go-qemu/qmp` is no longer compilable and gives the error:

```
# github.com/digitalocean/go-qemu/qmp
../../digitalocean/go-qemu/qmp/rpc.go:62:22: rpc.l.Events undefined (type *libvirt.Libvirt has no field or method Events)
```

I suspect its since this commit https://github.com/digitalocean/go-libvirt/commit/e2a69bcd5bd163f61a7e222cf915101e7e11b1c6

I've opened a similar issue at https://github.com/digitalocean/go-libvirt/issues/115, although the commit suggests an API breakage was expected.

--- a/qmp/rpc.go
+++ b/qmp/rpc.go
@@ -15,6 +15,7 @@
 package qmp
 
 import (
+	"context"
 	"encoding/json"
 	"net"
 
@@ -59,7 +60,7 @@ func (rpc *LibvirtRPCMonitor) Disconnect() error {
 // an error will be returned. Errors encountered during streaming will
 // cause the returned event channel to be closed.
 func (rpc *LibvirtRPCMonitor) Events() (<-chan Event, error) {
-	events, err := rpc.l.Events(rpc.Domain)
+	events, err := rpc.l.SubscribeQEMUEvents(context.Background(), rpc.Domain)
 	if err != nil {
 		return nil, err
 	}

